### PR TITLE
Visual redesign

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: breadcrumbs.scss [hunk 1]
```diff
@@ -1,11 +1,12 @@
-
 @import 'breadcrumbs-icons';
 @import 'bower_components/vui-link/link';
```
Comment (dbatiste) [inline breadcrumbs.scss]: We should update this to be more explicit too.  :)

File: breadcrumbs.scss [hunk 2]
```diff
@@ -24,7 +25,9 @@
 			}
 		}
 		&:after {
+			display: inline-block;
 			content: $vui-separator;
+			height: 1rem;
 			padding-left: 10px;
```
Comment (dbatiste) [inline breadcrumbs.scss]: Convert these to `rem`?

File: karma.conf.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ module.exports = function(config) {
     files: [
       { pattern: 'breadcrumbs.css', included: true },
```
Comment (dbatiste) [inline karma.conf.js]: I am partial to ditching these and setting up a test page (with Percy enabled) like we've done for a few repos.

File: test/perceptual/test.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```
Comment (dbatiste) [inline test/perceptual/test.html]: Ah - I see you have the `html` file for the percy test.  I think we should add build step to run, and update travis.yml.

File: karma.conf.js (hunk 3 above)
Comment (dlockhart) [inline karma.conf.js]: Totally agree here. Have a look at `vui-button` (including [my latest PR](https://github.com/Brightspace/valence-ui-button/pull/56)) for a good example of what to do there.

File: separator-rtl.svg [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
+  <title>
+    icon_chevron_left_18
+  </title>
```
Comment (dlockhart) [inline separator-rtl.svg]: You can ditch the titles, not needed.

File: separator.svg [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
+  <title>
+    icon_chevron_right_18
+  </title>
```
Comment (dlockhart) [inline separator.svg]: Ditto on the title.

File: test/perceptual/test.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+<html>
+<head>
+	<title>Breadcrumbs Sample</title>
+	<link rel="stylesheet" type="text/css" href="../../breadcrumbs.css">
```
Comment (dlockhart) [inline test/perceptual/test.html]: This also doesn't really test the mixins (although it does indirectly I guess). Again, have a look at button, input or field for examples of how that works. Or you can just merge this and I'll update it to be consistent with them as part of my documentation task.

Comment (dbatiste) [inline test/perceptual/test.html]: I think the ones that I did tend to test the mixins indirectly.

File: karma.conf.js (hunk 3 above)
Comment (njostonehouse) [inline karma.conf.js]: Cool! I'll de-Karma and en-Percy this repo after I release the visual redesigns.

**Pull request merged**